tests/ctst/common/hooks.ts

```diff
@@ -37,8 +49,129 @@ Before({ tags: '@Quotas', timeout: 1200000 }, async function (scenarioOptions) {
     await prepareQuotaScenarios(this as Zenko, scenarioOptions);
 });
 
+After(async function (this: Zenko, results) {
+    if (results.result?.status === 'FAILED') {
+        return;
```

Comment:
can we add a log here, so it is easy to see that something was not cleand up, and to make an explicit link between a specific test and what bucket/account/... were used (and retained in DB)